src/index.js

```diff
@@ -0,0 +1,91 @@
+let new_todo = document.getElementById('new-todo-title');
+let answer;
+
+new_todo.addEventListener("keyup", function(event) {
+    // Enter키를 누를시 실행
+    if (event.keyCode === 13) {
+        answer = confirm('등록 하시겠습니까?') == true ? admin(new_todo.value) : false;
+        //입력한 할일 추가시 할일 공백으로 만들기
+        new_todo.value = '';
+    }
+});         // 내용을 입력하고 'Enter'를 누르면 comfirm이 나오고 맞다면 admin함수 실행
+
+let count = 0 ;     //할 일 갯수 세는 용
+let todo_count = document.querySelector('.todo-count');
+
+// ul태그 추가
+let ul = document.createElement('ul');
+// main 태그를 기준으로 바로위에다가 ul 엘리먼트를 추가
+document.querySelector('main').insertAdjacentElement("beforebegin", ul);
+
+function admin(value) {
+    //태그들과 class 속성 부여 후 연결
+
+    let li = document.createElement('li');
+    let div = document.createElement('div');
+    let input = document.createElement('input');
+    let button = document.createElement('button');
+    let label = document.createElement('label');
+
+    ul.append(li);
+    li.append(div);
+    div.append(input);
+    div.append(label);
+    div.append(button);
+
+    ul.classList.add('todo');
+    ul.setAttribute('id','todo');
+    li.classList.add('active')
+    div.classList.add('new-todo');
+    input.classList.add('toggle');
+    input.setAttribute('type', 'checkbox');
+    label.classList.add(`label`);
+    button.classList.add(`destroy${count++}`);
+
+    //li앞에 문자 생략
+    li.style.listStyleType = "None";
+    //입력 값을 label에 추가 및 버튼에 X 추가
+    label.innerHTML = value;
+    button.innerHTML = 'X';
+
+    //버튼 클릭시 가장가까운 li태그 삭제 및, 해야할 count 값 줄이기
+    button.addEventListener('click', function (event) {
+        event.target.closest('li').remove();
+        todo_count.childNodes[1].innerHTML = --count;
```

Comment:
이 부분은 코드를 봤을 때 어떤 부분의 ```innerHTML```를 변경하려고 하는건지 파악하기 어렵습니다. 

(1) 상단에서 선언을 할 때 childNodes의 값까지 ```todo_count```에 할당해주시거나 
(2) ```querySelector```를 이용해서 ```.todo-count``` 요소 안에 있는 ```<strong>```태그만을 파싱해오면 더 좋을거 같습니다!

```suggestion
        let todo_count = document.querySelector('.todo-count');  // --- (1)
        let todo_count = document.querySelector('.todo-count > strong');) // --- (2)
        todo_count.innerHTML = --count;
 ```